### ISSUE TITLE
"Include Businesses" in Statement Generator Only available when not using a DV

### ISSUE THREAD

**Issue by rutledgek** [SparkDevNetwork/Rock#2720]
### Steps to Reproduce

1. Create A DV of transactions by businesses
2. Open Statement generator and select DV 
3. Businesses are not included 

**Expected behavior:** 

When selecting a DV, You would expect all entities in the DV to be included in the statement generator report.  

**Actual behavior:** 

Statement Generator is filtering DV in only looking for people.  The option to include Bussinesses is only available when looking at all individuals with transactions.

### Versions

* **Rock Version:** Rock McKinley 6.10 
* **Client Culture Setting:** en-US

**Comment by mr9802** [comment]
I am seeing the same thing.  Thanks Kevin!

**Comment by MikeDPeterson** [comment]
Note: For v6.10, this is fixed by updating from the Rock Shop

**Comment by mr9802** [comment]
Hey mike, I already updated them from the Rock Shop and the people DV work fine. It’s only when I select businesses that it does not work. I tried multiple different DV’s with businesses and none work. It appears at the end of the statement generator it is filtering for records with person.RecordType =1 for person. Make sense?  I have tried this on 3 rock installs all on 6.10 with the newest statement generator and plug-in installed.
